file-formats/aifd/type/zif_aff_aifd_v1.intf.abap

```diff
@@ -0,0 +1,41 @@
+INTERFACE zif_aff_aifd_v1
+  PUBLIC.
+  TYPES:
+    "! <p class="shorttext">AIF Interface</p>
+    "! AIF interface
+    "! $required
+    BEGIN OF ty_aif_interface,
+      "! <p class="shorttext">Interface Object Name</p>
+      "! Interface Object Name
+      "! $required
+      object_name       TYPE c LENGTH 40,
+      "! <p class="shorttext">Namespace</p>
+      "! Namespace
+      namespace         TYPE c LENGTH 6,
+      "! <p class="shorttext">Interface Name</p>
+      "! Interface Name
+      interface_name    TYPE c LENGTH 10,
+      "! <p class="shorttext">Interface Version</p>
+      "! Interface Version
+      interface_version TYPE c LENGTH 5,
+    END OF ty_aif_interface,
```

Comment:
Sorry, I am not an expert in AIF. What is the difference between an interface object name and an interface name?

Reply:
The interface object name is the technical key in the TADIR for the AIFI (AIF Interface) development object.
The AIF Interfaces consists of Namesapce, Interface Name and Interface Version.
Example:
      "objectName": "/AIFTCUST_WS_I00001",
      "namespace": "/AIFT",
      "interfaceName": "CUST_WS_I",
      "interfaceVersion": "00001"